react/package.json

```diff
@@ -2,69 +2,66 @@
   "name": "5calls",
   "version": "0.1.0",
   "private": true,
-  "engines": {
```

Comment:
I believe this is [largely meant for js libraries rather than application code](https://docs.npmjs.com/cli/v11/configuring-npm/package-json#engines)-- it seemed to be confusing parcel into trying to generate a library bundle.

Reply:
👍 I think I added this so it would remind me that I was using the wrong node version so fine to remove if it was causing unintended side effects